### PR TITLE
docs(mesh): add prod usage helm values

### PR DESCRIPTION
**Pull request by jijiechen**
### Description

 

fixes https://github.com/Kong/docs.konghq.com/issues/7507

replaces https://github.com/Kong/docs.konghq.com/pull/7508

### Testing instructions

Preview link: 

[2.8.x](https://deploy-preview-7640--kongdocs.netlify.app/mesh/2.8.x/production/cp-deployment/production-usage-values/)
[dev - 2.9](https://deploy-preview-7640--kongdocs.netlify.app/mesh/dev/production/cp-deployment/production-usage-values/)

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

**Comment by jijiechen** [comment]
The content accuracy was discussed and approved in this doc of the mesh repo:
https://github.com/Kong/kong-mesh/pull/5892

**Comment by jijiechen** [comment]
This PR replaces the previous one at https://github.com/Kong/docs.konghq.com/pull/7508
All suggestions as of July 15 are fixed.

File: app/_src/mesh/production/cp-deployment/production-usage-values.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Instructions on this page are meant to be the starting point for your installation. It's always recommended to read the full [Helm configuration reference](/mesh/{{ page.release }}/reference/kuma-cp/#helm-valuesyaml) to find support for other options available before executing the install steps. Values provided here will override values in the default reference Helm configuration, and they may be overridden again by further `values.yaml` files or arguments of the `helm` command. See the [Helm documentation](https://helm.sh/docs/chart_template_guide/values_files/) to learn more about how values are calculated.
+
+Values on this page may reference resources that need to be created in advance when certain features are enabled, read the file content carefully and prepare these resources according to the notes near the keywords `(action)`. If you decide to disable a feature that requires a pre-existing resource, remove or change those fields according to the full [Helm configuration reference](/mesh/{{ page.release }}/reference/kuma-cp/#helm-valuesyaml).
+
+
+{% navtabs %}
+{% navtab Single zone control plane %}
+
+## Install a single zone control plane
```
Comment (lena-larionova) [inline app/_src/mesh/production/cp-deployment/production-usage-values.md:15]:
```suggestion
Install a single zone control plane:
```
Headers don't really work inside navtab elements.

File: app/_src/mesh/production/cp-deployment/production-usage-values.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Suggested `values.yaml` file:
+
+```yaml
+{% embed helm-values-prod/values.single-zone-cp.yaml versioned %}
+```
+
+{% endnavtab %}
+{% navtab Multi-zone global control plane %}
+
+## Install the global control plane for a multi-zone deployment
```
Comment (lena-larionova) [inline app/_src/mesh/production/cp-deployment/production-usage-values.md:30]:
```suggestion
Install the global control plane for a multi-zone deployment:
```

File: app/_src/mesh/production/cp-deployment/production-usage-values.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+{% embed helm-values-prod/values.global-cp.yaml versioned %}
+```
+
+The values on this page may reference resources that need to be created in advance when certain features are enabled, read the file content carefully and prepare these resources according to the notes near the keywords `(action)`. If you decide to disable a feature that requires a pre-existing resource, remove or change those fields according to the full [Helm configuration reference](/mesh/{{ page.release }}/reference/kuma-cp/#helm-valuesyaml).
+
+{% endnavtab %}
+
+{% navtab Multi-zone federated zone control planes %}
+
+## Install federated zone control planes for a multi-zone deployment
```
Comment (lena-larionova) [inline app/_src/mesh/production/cp-deployment/production-usage-values.md:48]:
```suggestion
Install federated zone control planes for a multi-zone deployment:
```

**Review by lena-larionova — APPROVED** [review]
Minor edit on the headers, as they don't function properly inside navtab elements. Just need to switch them to regular text. Otherwise, LGTM

**Pull request merged**